outlines/routing.md

```diff
@@ -0,0 +1,40 @@
+# Routing
+
+* What is routing, and how it's different in a client-rendered app
+    * Uses for the URL
+        * Bookmarking
+        * Sharing
+        * Back/forward
+    * It's a useful serialization of some share-able, bookmarkable client-side state
+        * Not everything needs to be a route - consider that you can just store JavaScript variables, or use local storage for temporary information
+    * Routing is not related to data loading or authorization in a client-side app. For those, go see the security and data loading chapters
+* Flow Router intro
+* Creating routes
+    * How to define a basic route
+    * How to accept route parameters and URL pattern matching
+    * When to use query parameters vs. path parameters
```

Comment:
We don't necessarily need to cover this but it's nice to have a convention (i.e. rails uses `foo[]=7&foo[]=8`.